packages/shell-api/src/helpers.ts

```diff
@@ -429,12 +429,14 @@ export async function getPrintableShardStatus(db: Database, verbose: boolean): P
           collRes.balancing = [ !coll.noBalance, { noBalance: coll.noBalance } ];
         }
         const chunksRes = [];
+        const chunksCollMatch = coll.uuid ? { $or: [ { uuid: coll.uuid, ns: coll._id } ] } : { ns: coll._id };
         const chunks = await
-        (await chunksColl.aggregate({ $match: { ns: coll._id } },
+        (await chunksColl.aggregate([
+          { $match: chunksCollMatch },
           { $group: { _id: '$shard', cnt: { $sum: 1 } } },
           { $project: { _id: 0, shard: '$_id', nChunks: '$cnt' } },
-          { $sort: { shard: 1 } })
-        ).toArray();
+          { $sort: { shard: 1 } }
+        ])).toArray();
```

Comment:
The tests really should have discovered that `sh.status()` was also broken, I just noticed this because I was looking for other uses of `config.chunks` in the shell-api source. I’ve opened https://jira.mongodb.org/browse/MONGOSH-742 as a reminder to write proper tests here.